src/main/java/net/tardis/mod/client/TardisKeyBinds.java

```diff
@@ -0,0 +1,15 @@
+package net.tardis.mod.client;
+
+import net.minecraft.client.settings.KeyBinding;
+import net.minecraftforge.fml.client.registry.ClientRegistry;
+import net.tardis.mod.Tardis;
+import org.lwjgl.input.Keyboard;
+
+public class TardisKeyBinds {
+    public static KeyBinding DEMAT_REMAT_ANIM;
+
+    public static void init(){
+        DEMAT_REMAT_ANIM = new KeyBinding(Tardis.MODID+".keybinds.snap", Keyboard.KEY_J, Tardis.NAME);
```

Comment:
Incorrect Lang

Also add a lang entry

Reply:
Already done in local, but that's not my problem ! 😄 

Thanks for the review anyway ! :D